components/ProfileCardShareLink.vue

```diff
@@ -0,0 +1,101 @@
+<script setup lang="ts">
+import { useElementSize } from '@vueuse/core'
+
+type Props = {
+  profile?: Profile
+}
+
+const props = defineProps<Props>()
+const buttonRef = ref()
+const buttonWidth = ref('auto')
+const buttonFullWidth = ref('auto')
+const hover = ref(false)
+const isCopied = ref(false)
+const { currentNetwork } = storeToRefs(useAppStore())
+
+const handleCopy = () => {
+  navigator.clipboard.writeText(linkUrl.value)
+  isCopied.value = true
+
+  setTimeout(() => {
+    hover.value = false
+    isCopied.value = false
+  }, 500)
+}
+
+const linkUrl = computed(() => {
+  if (props.profile?.profileLink?.isResolved) {
+    return props.profile?.profileLink?.link || ''
+  }
+
+  // if we can't resolve we just link to the wallet
+  return `${BASE_WALLET_URL}/${props.profile?.address}/?network=${currentNetwork.value.id}`
```

Comment:
not sure this works :/
<img width="1146" alt="image" src="https://github.com/lukso-network/wallet.universalprofile.cloud/assets/32359115/2202508b-3c81-4da0-99bd-3a0a3264b974">

Reply:
@richtera any idea?

Reply:
ah the last commit didn't pass the check, that's why